.github/workflows/ci-aarchxx.yml

```diff
@@ -209,6 +209,7 @@ jobs:
         cd /tmp
```

Comment:
About the PR title: let's try to change it to something like the following shorter version, so that it doesn't get wrapped by Github UI. 

`i#5527: fix android build broken by preset ANDROID_NDK_ROOT`

The remaining details can then be included in the PR description instead

Reply:
Done.